build/common.sh

```diff
@@ -1235,3 +1235,8 @@ list_packages()
 
 	list_config ${@}
 }
+
+multi_line_list()
+{
+	echo "${@}" | tr ',[:blank:]' '\n'
```

Comment:
the old function had a for loop which is softer to the eye and does auto-conversion of whitespace separated entities. but in any case this is so generic that it hardly makes sense to unify.. old code folded all in list_config/list_packages (or it's former equivalents)